sympy/polys/ring_series.py

```diff
@@ -572,6 +574,111 @@ def _coefficient_t(p, t):
             p1[monomial_div(expv, expv1)] = p[expv]
     return p1
 
+def rs_fast_series_reversion(p,x,n):
+
+    r"""
+    Fast series reversion (aka Lagrange Inversion) using Newton-like updates.
+
+    Given a ring polynomial $p$ in variable $x$, this method computes a ring polynomial $q$ in $x$ such that
+    $q(p(x)) = x + O(x^n)$.
```

Comment:
Long lines should be wrapped.

Reply:
@oscarbenjamin this should have been checked by formatters. To avoid manual checks as yours. 
How about adding such a rule?